### DISCUSSION TITLE
Migrazione da duckdns a cloudflare e Haaska

### DISCUSSION THREAD

**Discussion by JackTalisker** [indomus/forum#2828] [HUB: Home Assistant]
Buongiorno a tutti,
il mio sistema HA Core  in container docker sta attualmente funzionando bene utilizzando Caddy come reverse proxy con un FQDN fornito da Duckdns; anche Haaska è configurato e funziona come deve.

Per una serie di motivi ho deciso di migrare a Cloudflare utilizzando un mio dominio su Aruba.
Una volta che terminerò la migrazione (con l'utile contributo delle vostre guide) ho bisogno di adeguare la configurazione di Haaska.

La domanda è: 
sarà sufficiente sostituire nel file "config.json" (del codice Haaska) la vecchia URL  "url": "https://cippalippa.duckdns.org:8123/api", con la nuova  "url": "https://miodominio.suAruba.it/api" e rifare il deploy?
o c'è qualcos'altro da adeguare che mi sfugge?

Grazie a tutti.

**Comment by snakuzzo** [comment]
Corretto.

Ribatti la guida però, giusto per vedere se ci sono altri punti in cui si fa riferimento al proprio dominio

**Reply by JackTalisker** [reply]
No, in effetti ho controllato e non sembrano esserci altri riferimenti.
Ovviamente immagino che il token creato a suo tempo sia valido e non vada ricreato.

Grazie!

**Reply by snakuzzo** [reply]
Esatto...il token resta lo stesso
